elevate-cpanel

```diff
@@ -1148,6 +1150,26 @@ sub check_and_fix_grub ($self) {
     return unless $grub_conf =~ s/GRUB_CMDLINE_LINUX="(.+?)"/GRUB_CMDLINE_LINUX="$1 net.ifnames=0"/m;
     File::Slurper::write_binary( DEFAULT_GRUB_FILE, $grub_conf );
 
+    # Also save net.ifnames=0 to the GRUB environment:
+    my $grubenv = Cpanel::SafeRun::Simple::saferunnoerror( GRUB_EDITENV, GRUB_ENV_FILE, 'list' );
+    foreach my $line ( split "\n", $grubenv ) {
+        my ( $name, $value ) = split "=", $line, 2;
+        next unless $name eq "kernelopts";
+
+        if ( $value !~ m/\bnet\.ifnames=0\b/a ) {
```

Comment:
probably better to check for `net.ifnames=` and not the value?

Reply:
> probably better to check for `net.ifnames=` and not the value?

probably

Reply:
looks good except for this.

Reply:
Let's remove `0\b` from the regex and I think we're good.